detectron/utils/vis.py

```diff
@@ -129,7 +129,8 @@ def vis_class(img, pos, class_str, font_scale=0.35):
 
 
 def vis_bbox(img, bbox, thick=1):
-    """Visualizes a bounding box."""
+    """Visualizes a bounding box.""" 
```

Comment:
I think there is one more trailing space left here. Could you please delete it too? Sorry I missed it earlier.